pkgs/applications/taxes/aangifte-2006/default.nix

```diff
@@ -5,15 +5,15 @@ stdenv.mkDerivation {
   builder = ./builder.sh;
 
   src = fetchurl {
-    url = http://download.belastingdienst.nl/belastingdienst/apps/linux/ib2006_linux.tar.gz;
+    homepage = http://download.belastingdienst.nl/belastingdienst/apps/linux/ib2006_linux.tar.gz;
```

Comment:
this one isn't right

Reply:
Oh, aangifte-* were the only not done fully manually and they got wrong without me noticing :-/